Documentation/ceph-object-store-crd.md

```diff
@@ -89,7 +89,7 @@ The gateway settings correspond to the RGW daemon settings.
 
 * `type`: `S3` is supported
 * `sslCertificateRef`: If the certificate is not specified, SSL will not be configured. If specified, this is the name of the Kubernetes secret that contains the SSL certificate to be used for secure connections to the object store. Rook will look in the secret provided at the `cert` key name. The value of the `cert` key must be in the format expected by the [RGW service](https://docs.ceph.com/docs/master/install/ceph-deploy/install-ceph-gateway/#using-ssl-with-civetweb): "The server key, server certificate, and any other CA or intermediate certificates be supplied in one file. Each of these items must be in pem form."
-* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted).
+* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted) if host network is enabled. Otherwise, this will be the service port and RGW daemon will be hardcorded to 8080.
```

Comment:
It's not hardcoded, right? That would mean it's not configurable. But it is configurable in the rgw-external.yaml.

Reply:
@travisn The hard corded value of [rgwPortInternalPort](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/config.go#L42) is used in [generateLiveProbe](https://github.com/rook/rook/blob/master/pkg/operator/ceph/object/spec.go#L172) in the beginning of the function. This will change to `port` in object-store-crd only if host network is enabled. And I am trying to document this info here. This information does not matter in converged mode and but if I am trying to access rgw service externally, we need to provide `internal port` information properly in `rgw-external.yaml`. If we rely only on `service port` of rgw service  it won't work